common/src/envvars.ts

```diff
@@ -29,8 +29,13 @@ function getSubstitutionValue(regexMatch: string, placeholder: string): string {
 // In the latter case, the corresponding returned item would be "BAR=hi"
 // where the value is taken from the matching process env var.
 // In the case of values not set in the process, they are omitted
-export function populateDefaults(envs: string[]): string[] {
+export function populateDefaults(envs: string[], inerhitEnv: boolean): string[] {
 	const result: string[] = [];
+	if (inerhitEnv) {
+		for (const [key, value] of Object.entries(process.env)) {
```

Comment:
Should some variables be skipped? E.g., `PATH` might cause issues, maybe others.

Another approach would be to make it easy to copy over specific variables. What are the main usages for this? All CI env variables with a common prefix?

Reply:
I'm happy to have a reject list for troublesome variables, like `PATH`, but the feature is specifically about variables from dependencies/other actions, that are not explicitly specified/controlled by the user.
Please check the discussion on #242 for more info/examples.

Reply:
Added `PATH` for now. Please let me know if I can add anything else.